lmod/openmpi_custom.lua

```diff
@@ -28,7 +28,7 @@ if posix.stat(pathJoin(slurmpath,"libslurm.so.36"),"type") == "link" then
 	setenv("OMPI_MCA_plm_slurm_args", "--whole")
 end
 
-if ompiv ~= "3.1" and ompiv ~= '4.0' and ompiv ~= '4.1' then
+if ompiv ~= "3.1" and ompiv ~= '4.0' and ompiv ~= '4.1' and ompiv ~= '5.0' then
```

Comment:
This starts feeling like we should reverse the check, i.e. check only for older version, rather than keep excluding new versions

Reply:
done